api-server/src/graphql/schema.graphql

```diff
@@ -399,6 +439,15 @@ type JobApplication {
 	fullName: String!
 	email: String!
 	phoneNumber: String!
+	city: String!
+	neighborhood: String
+	workExperiences: [WorkExperience]
+	skills: [String]!
+	certificatesAndLicences: [String]
+	englishProficiency: String!
+	highestLevelOfEducation: String!
```

Comment:
`englishProficiency` and `highestLevelOfEducation` could use enum types instead of strings to make the API more clear. For example look at `ContractType` and `EmploymentStatus`.